scripts/lib/CIME/tests/scripts_regression_tests.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
+
+"""
+Script containing CIME python regression test suite. This suite should be run
+to confirm overall CIME correctness.
+"""
+
+import glob, os, re, shutil, signal, sys, tempfile, \
```

Comment:
- [ ] A lot of the imports in this file are no longer referenced.

Reply:
The plan is resolve this in my next PR that'll be targeting code formatting/style enforcement. I didn't want to add extra changes to this PR and rather tackle it all in a separate one.

Reply:
Sounds good - thanks.